### ISSUE TITLE
Proposal: Optimise high level structure for target audience

### ISSUE THREAD

**Issue by michaelstingl** [owncloud/docs#695]
I was thinking about an approach to optimise the structure for the different target audiences. This is first draft:
(Examples in brackets)

---

### User Documentation

- Web UI/Client
- Desktop Sync Client (Installation, usage, easy troubleshooting)
- Android App
- iOS App

### Admin Documentation

- ownCloud Server Backend (Installation, Configuration, Maintenance)
- Web UI/Client (Branding)
- Desktop Sync Client (Branding, Mass Deployments, more troubleshooting)
- Android App (Branding, Play Store Distribution, MDM)
- iOS App (Branding, App Store Distribution, MDM)

### Developer Documentation

- ownCloud Server Backend (PHP API, OCS API, manual theming, app development)
- Web UI/Client (ownCloud Design System)
- Desktop Sync Client (Build the client)
- Android App (Build the app, use SDK)
- iOS App & SDK (Build the app, use SDK)

### ownCloud Product Documentation

- Platform (Basic principles across all platforms)
- Web UI/Client (UI/Design)
- Desktop Sync Client
- Android App
- iOS App
- ownCloud Server Backend

---

## Where Does This Need To Be Documented?

This would be the root level for all other docs.

## Why Should This Change Be Made? (Optional)

Optimise the structure for the different target audiences.

## What Type Of Content Change Is This? (Optional)
- [ ] New Content Addition
- [ ] Old Content Deprecation
- [x] Existing Content Simplification
- [ ] Bug Fix to Existing Content

## Which Manual Does This Relate To? (Optional)
- [x] Admin Manual
- [x] Developer Manual
- [x] User Manual
- [x] Android
- [x] iOS
- [x] Branded Clients
- [x] Desktop Client
- [x] Other

We would need to split up the desktop and mobile docs for the different audiences. This can be done over time…

@settermjd @pmaier1 @felixboehm I talked with you in the past – what do think? Please provide feedback, and I'll iterate…

**Comment by michaelstingl** [comment]
Related issues with requests to change the structure:
- https://github.com/owncloud/docs/issues/302 (mostly Server Admin Manual)
- https://github.com/owncloud/docs/issues/658 (mostly Install instructions in Server Admin Manual; closed for now – will be handled in follow-up issues)

Those change requests are on a higher level and can be done independent from my proposal.

**Comment by pmaier1** [comment]
Nice, thanks! I like the structure, makes a lot more sense than the current one and should guide the respective audiences better :+1: 

Would we then drop the versions (as we currently have 10.0, 9.1, 9.0, etc.) and only have one documentation for the current version?

**Comment by felixboehm** [comment]
Basically, this is an overview page of all the different docs.
And versions are one more dimension on top...

By default, just show the current version, with „latest“ in the path.

**Comment by michaelstingl** [comment]
Yes, for every product you can have multiple versions. Antora navigation can handle this. But you don't need to list all previous versions on the overview page. That would ruin it.

**Comment by settermjd** [comment]
I really like where this is going, @michaelstingl. Perhaps we should catch up and map it out further. Then, I can work with @PVince81 to get it into a future sprint, so that we can begin to implement it.

**Comment by michaelstingl** [comment]
@settermjd I think we don’t need changes in the repo structure for the server backend. Or should we bring the server docs to the server repo?

For the client/apps repos, I think we only need subfolders? User, Admin, Developer?

Does this work for Antora?

**Comment by settermjd** [comment]
@PVince81, can we get time for this scheduled, pls?

**Comment by PVince81** [comment]
we have some tasks to finish currently, can schedule this afterward, yes

**Comment by voroyam** [comment]
@michaelstingl I would love to get together and exchange concrete plans for restructuring the user docs.

**Comment by voroyam** [comment]
2 years without progress.

**Comment by DeepDiver1975** [comment]
**Classification:** invalid-fixed (enhancement already implemented)

Thanks for this proposal and the thoughtful discussion. Reviewing the current state of `owncloud/docs`, the high-level restructure proposed here has effectively been carried out: the repository is now the **ownCloud Documentation (v2)** site, built on **Antora**, with per-product documentation and per-version navigation (defaulting to the current/`latest` version) — exactly the audience-oriented, multi-product structure discussed in this thread (e.g. showing only the current version on overview pages, Antora handling the version dimension).

Since the documentation has since migrated to the v2 Antora-based structure that this proposal advocated, we are closing this as implemented/obsolete.

If you believe a specific aspect of the proposed structure is still missing on the current docs site, please open a fresh issue with concrete, up-to-date details so it can be tracked against the current setup — thank you\!

🤖 Automated issue analysis by Claude Code review agent.
